osbenchmark/workload_generator/workload_generator.py

```diff
@@ -108,35 +101,80 @@ def process_custom_queries(custom_queries):
 
     return data
 
+def write_template(template_vars, templates_path, output_path, template_file):
+    template = get_template(template_file, templates_path)
+    with open(output_path, "w") as f:
+        f.write(template.render(template_vars))
+
+def get_template(template_file, templates_path):
+    template_file_name = template_file  + ".json.j2"
+
+    env = Environment(loader=FileSystemLoader(templates_path), autoescape=select_autoescape(['html', 'xml']))
+
+    return env.get_template(template_file_name)
+
+def render_templates(
+    workload_path, operations_path, test_procedures_path, templates_path, template_vars, custom_queries
+    ):
```

Comment:
Personally prefer formatting like:
```
def render_templates(workload_path,
                     operations_path,
```
since it "looks" more like a function, but feel free to disagree.